common/api-review/auth-exp.api.md

```diff
@@ -309,6 +310,11 @@ export class PhoneMultiFactorGenerator implements externs.PhoneMultiFactorGenera
     static assertion(credential: externs.PhoneAuthCredential): externs.PhoneMultiFactorAssertion;
 }
 
+// Warning: (ae-forgotten-export) The symbol "AuthErrorMap" needs to be exported by the entry point index.d.ts
+//
+// @public (undocumented)
+export const prodErrorMap: AuthErrorMap;

```

Comment:
why is this getting referenced in the docs?

Reply:
Should be better now